### PR TITLE
docs: add FAQ about TF2

### PR DESCRIPTION
**Pull request by aaron276h**

### CODE REVIEW

File: docs/faq.txt [hunk 1]
```diff
@@ -201,6 +201,20 @@ model to use :ref:`Estimator Trial <estimator-trial>`. `Example of
 converting a Tensorflow graph into an Estimator
 <https://github.com/determined-ai/determined/blob/master/examples/computer_vision/mnist_tf_layers/model_def.py>`_.
 
+Can I train using Tensorflow 2 in Determined?
+=============================================
+
+Yes Determined support both Tensorflow 1 and 2. The version of
```
Comment (neilconway) [inline docs/faq.txt:207]: "Yes, Determined supports..."

File: docs/faq.txt [hunk 2]
```diff
@@ -201,6 +201,20 @@ model to use :ref:`Estimator Trial <estimator-trial>`. `Example of
...
+Tensorflow used is configured by the version of Tesnsorflow that is
```
Comment (neilconway) [inline docs/faq.txt:208]: TensorFlow

File: docs/faq.txt [hunk 3]
```diff
@@ -201,6 +201,20 @@ model to use :ref:`Estimator Trial <estimator-trial>`. `Example of
...
+installed in docker image used for training. Determined offers pre-built
```
Comment (neilconway) [inline docs/faq.txt:209]: "Docker"

"the Docker image"

File: docs/Makefile [hunk 4]
```diff
@@ -12,7 +12,7 @@ build-helm:
 	$(MAKE) -C ../helm build
 
 reference/attributions.txt: $(shell find ../tools/scripts/licenses -type f)
-	../tools/scripts/gen-attributions.py sphinx > $@
+	python ../tools/scripts/gen-attributions.py sphinx > $@
```
Comment (neilconway) [inline docs/Makefile:15]: Why this change?

Comment (aaron276h) [reply docs/Makefile]: ah this snuck it, was having issues building docs

File: docs/faq.txt [hunk 5]
```diff
@@ -201,6 +201,20 @@ model to use :ref:`Estimator Trial <estimator-trial>`. `Example of
 converting a Tensorflow graph into an Estimator
 <https://github.com/determined-ai/determined/blob/master/examples/computer_vision/mnist_tf_layers/model_def.py>`_.
 
+Can I train using Tensorflow 2 in Determined?
```
[the remark this answers is not in the document]
Comment (aaron276h) [reply docs/faq.txt]: done

File: docs/faq.txt (hunk 1 above)
Comment (aaron276h) [reply docs/faq.txt]: done

File: docs/faq.txt (hunk 3 above)
Comment (aaron276h) [reply docs/faq.txt]: done

**Comment by neilconway** [comment]
@aaron276h I pushed some fixes to your branch.

**Review by neilconway — APPROVED** [review]
